BGPViewerCore/Service/BGPHeService.cs

```diff
@@ -338,15 +350,7 @@ public IpDetailModel GetIpDetails(string ipAddress)
                 ipDetails.PtrRecord = hasPtrRecord ? Regex.Match(ipAndPtrRecord, @"\(([^\)]+)\)").Value.Trim('(', ')') : null;
             }
             
-            ipDetails.RelatedPrefixes = ipInfoElement.FindElement(By.TagName("table"))
-                .FindElement(By.TagName("tbody"))
-                .FindElements(By.TagName("tr"))
-                .Select(tr => tr.FindElements(By.TagName("td")))
-                .Select(tds => new string[]{
-                    tds.ElementAt(0).FindElement(By.TagName("a")).GetAttribute("innerHTML").Substring(2), // AS number
-                    tds.ElementAt(1).FindElement(By.TagName("a")).GetAttribute("innerHTML"), // Prefix
-                    tds.ElementAt(2).GetAttribute("innerHTML") // Name/Description
-                })
+            ipDetails.RelatedPrefixes = ExtractAsDataFromIpInfoElement(ipInfoElement)
```

Comment:
ExtractAsDataFromIpInfoElement() could return an object with AsNumber, Prefix and Name properties instead of an string array. This way it'll be easier to understand its response.